benchmark/Microsoft.IdentityModel.Benchmarks/Microsoft.IdentityModel.Benchmarks.csproj

```diff
@@ -1,8 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <AssemblyName>Microsoft.IdentityModel.Benchmarks</AssemblyName>
```

Comment:
has this been added to the .sln so it shows up in a Benchmarks folder in VS when opening wilson.sln?